src/DataTypes/Serializations/SerializationBool.cpp

```diff
@@ -332,4 +332,10 @@ void SerializationBool::serializeTextXML(const IColumn & column, size_t row_num,
     serializeSimple(column, row_num, ostr, settings);
 }
 
+void SerializationBool::serializeTextMarkdown(
+    const DB::IColumn & column, size_t row_num, DB::WriteBuffer & ostr, const DB::FormatSettings & settings) const
+{
+    serializeTextEscaped(column, row_num, ostr, settings);
```

Comment:
That's the same as in `ISerialization` as the base class implementation, right? Why do we need it then? There are similar places in the code, e.g. for Dates...